.github/workflows/functional_tests.yaml

```diff
@@ -136,6 +136,20 @@ jobs:
           MVPN_API_BASE_URL: http://localhost:5000
           MVPN_BIN: ./build/mozillavpn
 
+      - name: Generate allure results
+        uses: actions/upload-artifact@master
+        with:
+          name: allure-results
+          path: allure-results
+          retention-days: 20
+
+      - name: Deploy report to github pages
```

Comment:
This will would cause each task (so for each test file) to publish to gh-pages, which we're currently using for wasm and the inspector -  Probably overwriting our current deployments. Not sure whether this is what you want  😅

Reply:
I would suggest using the `destination_dir` to pick a unique subdir for the branch and have a 2nd task on merge that will clean that up

Reply:
lol thanks for looking at it, but no its not, still draft, pushing up some changes soon